gdc_client/parcel/segment.py

```diff
@@ -75,13 +66,10 @@ def _setup_work(self):
         self.block_size = work_size // self.n_procs
 
     def _setup_queues(self):
-        if WINDOWS:
-            self.q_work = Queue()
-            self.q_complete = Queue()
-        else:
-            manager = Manager()
-            self.q_work = manager.Queue()
-            self.q_complete = manager.Queue()
+
+        manager = Manager()
+        self.q_work = manager.Queue()
+        self.q_complete = manager.Queue()
```

Comment:
I think the previous implementation used `Manager` only on posix systems due to windows lack of support for `fork()`. Not sure if this breaks anything but we should at least try it out on a windows system (that is if you have not already) before merging

Reply:
I download the source code and run in windows 10 with python 3.6. All the tests passed. 

To get the test passed, I have to modify the conftest.py file to use threading instead of Process from multiprocessing because it would try to pick the flask app run, which will fail. The Thread would never stop. Because there is no easy way to terminate it. 

There are 2 ways to solve this problem:
1. Some said on Stackoverflow that py3.7 will solve this
2. Pathos package use dill instack of pickle, which should work.

Reply:
I double-checked the test coverage. This part is not covered by any test. 
I run in debug mode manually and try to download a 100MB+ file and the modified version works for python 3.6.